node/ltnodeclient.js

```diff
@@ -93,10 +93,12 @@ function sbRequire(require, resolve, path) {
   if(!getSB(path)) {
     if(path.match(/[\\\/]/) && fpath.extname(path) == ".js") {
       var sb = sandboxes[path.toLowerCase()] = context(path);
```

Comment:
Ok. Thanks.
And so.. Do you need a maintainer for this repo? I would be happy to join you as maintainer for the JavaScript plugin or even for the main or the other side projects. I search to invest myself in OSS joining maintainer teams and GitHub organizations for the projects I love. And Light Table is becoming one of them ;-)